compose/config/config.py

```diff
@@ -198,20 +197,19 @@ def version(self):
         version = self.config['version']
 
         if isinstance(version, dict):
-            log.warn('Unexpected type for "version" key in "{}". Assuming '
-                     '"version" is the name of a service, and defaulting to '
-                     'Compose file version 1.'.format(self.filename))
+            log.warning('Unexpected type for "version" key in "{}". Assuming '
+                        '"version" is the name of a service, and defaulting to '
+                        'Compose file version 1.'.format(self.filename))
             return V1
 
         if not isinstance(version, six.string_types):
             raise ConfigurationError(
-                'Version in "{}" is invalid - it should be a string.'
-                .format(self.filename))
+                'Version in "{}" is invalid - it should be a string.'.format(self.filename))
 
         if version == '1':
             raise ConfigurationError(
                 'Version in "{}" is invalid. {}'
-                .format(self.filename, VERSION_EXPLANATION)
+                    .format(self.filename, VERSION_EXPLANATION)
```

Comment:
Please undo this indentation, since it's not related to the PR. It also triggers:
> PEP 8: continuation line unaligned for handling indent